### PR TITLE
changes: 'managing dependencies' section

### PR DESCRIPTION
**Pull request by shaunlebron**
(leaving comments by the relevant lines)

### CODE REVIEW

File: src/compiler.adoc [hunk 1]
```diff
@@ -779,9 +779,8 @@ will be familiar with almost all the scripts that comes with *mies* template.
 
 ==== Managing dependencies
 
-The real purpose of use leiningen for the ClojureScript compilation process is managing
-dependencies. Because it makes this task more obvious in comparison to doing it
-manually.
+The real purpose of using Leiningen for the ClojureScript compilation process is to automate
+the retrieval of dependencies.  This is dramatically simpler than retrieving them manually.
```
Comment (shaunlebron) [inline src/compiler.adoc]: - verb tense: of use -> of using
- elaborate: manage dependencies -> automate retrieval of dependencies

File: src/compiler.adoc [hunk 2]
```diff
@@ -812,7 +811,7 @@ The dependencies in ClojureScript are packaged using `jar` files. If you are com
 Clojure or any JVM language, `jar` files will be very familiar to you. But if you
 aren't familiar with them, do not worry: a .jar file is like a plain zip file that contains
 the `project.clj` for the library, some metadata, and the ClojureScript sources. The
-packaging will be explained in other section.
+packaging will be explained in another section.
```
Comment (shaunlebron) [inline src/compiler.adoc]: - idiom: other -> another

File: src/compiler.adoc [hunk 3]
```diff
@@ -821,7 +820,7 @@ link:https://github.com/clojure/clojurescript/wiki#libraries[ClojureScript Wiki]
 To understand dependencies better, we are going to replace the DOM utiltiles of the Google Closure
 library with a third party library for interacting with the DOM.
 
-Let start adding `[prismatic/dommy "1.1.0"]` into the `:dependencies` in the
+Let's start by adding `[prismatic/dommy "1.1.0"]` at the end of the `:dependencies` vector in
```
Comment (shaunlebron) [inline src/compiler.adoc]: - grammar: Let start adding -> Let's start by adding
- elaborate: where to add dependency so reader may see it immediately

File: src/compiler.adoc [hunk 4]
```diff
@@ -846,8 +845,9 @@ Building ...
 Watching paths: /home/niwi/cljsbook/playground/leapyears/src
 ----
 
-You can observe that dommy dependency is successfully downloaded, and then the watch
-process is started.
+You can observe that the dommy dependency is successfully downloaded, and then the
+watch process is started.  In fact, Leiningen will ensure all dependencies are
+downloaded before performing any tasks.
```
Comment (shaunlebron) [inline src/compiler.adoc]: - article: (+the) dommy dependency
- elaborate: explain when lein downloads dependencies

File: src/compiler.adoc [hunk 5]
```diff
@@ -881,10 +881,8 @@ with the following content:
 ----
 
 As you can observe, the code now looks more “Clojure-friendly.” The host
-interoperabilty calls are almost hidden.
-
-NOTE: This example does not means that we prefer `dommy` over the Google Closure library. In
-fact, dommy does not have documentation, so we do not recommend use it.
+interoperabilty calls have now been replaced by functions provided by this
+third-party library.
```
Comment (shaunlebron) [inline src/compiler.adoc]: - rephrase: hidden -> replaced with ... (just concludes the section by fully stating what we accomplished)
- opinion: removed opinion on dommy, thinking about impartiality

**Comment by niwinz** [comment]
\o/

**Pull request merged**